cmd/grpcurl/.goreleaser.yml

```diff
@@ -0,0 +1,22 @@
+builds:
+  - binary: grpcurl
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - 386
+
+archive:
+  wrap_in_directory: true
+  format: tar.gz
+  format_overrides:
+    - goos: windows
+      format: zip
+  replacements:
+    amd64: x64
```

Comment:
I was just looking at the `protoc` distribution, and they use `x86_64` as the architecture here and `x86_32` for `386`. Maybe do the same, for consistency with a closely related binary?

Reply:
💯 👍